dll/win32/shell32/wine/pidl.c

```diff
@@ -555,8 +555,11 @@ BOOL WINAPI ILIsEqual(LPCITEMIDLIST pidl1, LPCITEMIDLIST pidl2)
     while (pidltemp1->mkid.cb && pidltemp2->mkid.cb)
     {
         if (!_ILHACKCompareSimpleIds(pidltemp1, pidltemp2))
+#ifndef __REACTOS__
             return FALSE;
-
+#else
+            return TRUE;
+#endif
```

Comment:
So is this a hack on top of a hack?

Reply:
I'm pretty sure it's a bug. I'll run the ILIsEqual rostest to make sure

Reply:
This is a function that is pretty crucial to the inner workings of the shell, and a change like this is hard to guesstimate the impact.
It would help a lot if you can prove your changes with an apitest.